_tools/index.md

```diff
@@ -26,7 +26,7 @@ For information about Data Prepper, the server-side data collector for filtering
 
 Historically, many multiple popular agents and ingestion tools have worked with Elasticsearch OSS, such as Beats, Logstash, Fluentd, FluentBit, and OpenTelemetry. OpenSearch aims to continue to support a broad set of agents and ingestion tools, but not all have been tested or have explicitly added OpenSearch compatibility.
 
-As an intermediate compatibility solution, OpenSearch has a setting that instructs the cluster to return version 7.10.2 rather than its actual version.
+As an intermediate compatibility solution, OpenSearch 1.x and 2.x have a setting that instructs the cluster to return version 7.10.2 rather than its actual version.
```

Comment:
```suggestion
As an intermediate compatibility solution, OpenSearch 1.x and 2.x provide a setting that instructs the cluster to return version 7.10.2 rather than its actual version.
```